src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -568,6 +569,12 @@ private bool ProcessCssFile(IRequestInfo info, string localPath)
 			// if still not found, and localPath is an actual file path, use it
 			if (string.IsNullOrEmpty(path) && File.Exists(localPath)) path = localPath;
 
+			// check if this is a path in BloomBrowserUI
+			if (string.IsNullOrEmpty(path))
+			{
+				path = FileLocator.GetFileDistributedWithApplication("BloomBrowserUI", localPath);				
+			}
+
```

Comment:
Shouldn't we just configure the _fileLocator used above to know about this possible location?